excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/service/ImportService.java

```diff
@@ -8,7 +8,8 @@
 
 public interface ImportService {
 
-    DefinitionFileUploadMetadata importFormDefinitions(InputStream inputStream) throws IOException;
+    DefinitionFileUploadMetadata importFormDefinitions(InputStream inputStream, boolean reindex,
+                                                       boolean deleteOldIndex) throws IOException;
```

Comment:
instead of primitive boolen please use Boolean object

Boolean reindex,
Boolean deleteOldIndex